README.md

```diff
@@ -129,3 +129,659 @@ SLINGR is a low-code rapid application development platform that accelerates dev
 ## License
```

Comment:
According to the general format, we are leaving the "# About SLINGR" and "# License" sections at the end of the Readme.